python/tester/upload.py

```diff
@@ -217,7 +218,7 @@ def get_samples_non_lsst(bucket, instrument):
             dome=FannedOutVisit.Dome.OPEN,
             duration=float(EXPOSURE_INTERVAL+SLEW_INTERVAL),
             totalCheckpoints=1,
-            private_sndStamp=hsc_metadata[exp_id]["time"],
+            private_sndStamp=hsc_metadata[exp_id]["time"]-60,
```

Comment:
For self-consistency, it might be better to calculate the offset from `EXPOSURE_INTERVAL` and `SLEW_INTERVAL`.

Reply:
Good idea.  I'm setting them to 2*duration before the exposure start time.  All these timestamps were in the past, as from the original raws' metadata, so they will need to be better simulated to be useful.